packages/zoe/src/zoe.chainmail

```diff
@@ -122,29 +117,27 @@ interface ZoeGoverningContractFacet () {
    * done here because any previous reallocation performed those checks.
    */
   complete (offerHandles :List(Object));
+  
+  /** Create an invite using the Zoe inviteMint */
+  makeInvite (seat :Object, customProperties :Object ) -> (Payment);
 
-  /**
-   * Create an empty offer for recordkeeping purposes (Autoswap uses
-   * this to create the liquidity pool). 
+  /** 
+   * Inform Zoe about new assays. Returns a promise for acknowledging
+   * when the assays are added and ready.
    */
-  escrowEmptyOffer ( ) -> (offerHandle);
-  
+  addAssays (assays :List(Object));
```

Comment:
This is a promise for `undefined`. How do we say that in the method declaration? I think it needs to, since it is more meaningful (and later) than the normal promise-for-undefined for methods that do not explicitly return anything.

Reply:
I added a fix, let me know if you think it works